### PR TITLE
RD-4412 Add new tokens model

### PR DESCRIPTION
**Pull request by geokala**

### CODE REVIEW

File: rest-service/manager_rest/storage/management_models.py [hunk 1]
```diff
@@ -474,6 +481,19 @@ def is_locked(self):
         return False
 
 
+class Token(SQLModelBase, CreatedAtMixin):
```
Comment (tehasdf) [inline rest-service/manager_rest/storage/management_models.py:484]: expiration date? execution id?

File: rest-service/manager_rest/storage/management_models.py [hunk 2]
```diff
@@ -474,6 +481,19 @@ def is_locked(self):
...
+    __tablename__ = 'tokens'
+
+    id = db.Column(db.String(10), primary_key=True, index=True)
+    secret = db.Column(db.String(255), nullable=False)
```
Comment (tehasdf) [inline rest-service/manager_rest/storage/management_models.py:488]: string? maybe binary?

File: resources/rest-service/cloudify/migrations/versions/272e61bf5f4a_6_3_to_6_4.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    op.create_index(op.f('tokens_created_at_idx'),
+                    'tokens', ['created_at'], unique=False)
+    op.create_index(op.f('tokens_id_idx'), 'tokens', ['id'], unique=True)
+
+
+def downgrade():
+    op.drop_index(op.f('tokens_last_used_idx'), table_name='tokens')
+    op.drop_index(op.f('tokens_id_idx'), table_name='tokens')
+    op.drop_index(op.f('tokens_created_at_idx'), table_name='tokens')
+    op.drop_table('tokens')
```
Comment (tehasdf) [inline resources/rest-service/cloudify/migrations/versions/272e61bf5f4a_6_3_to_6_4.py:45]: might as well just drop the table only, i mean, indexes will be gone too

File: rest-service/manager_rest/storage/management_models.py (hunk 2 above)
Comment (tehasdf) [reply rest-service/manager_rest/storage/management_models.py:488]: if it's a string, i assume it's base64?

Comment (geokala) [reply rest-service/manager_rest/storage/management_models.py:488]: It's the same as the password field on the users table.

File: resources/rest-service/cloudify/migrations/versions/272e61bf5f4a_6_3_to_6_4.py (hunk 3 above)
Comment (geokala) [reply resources/rest-service/cloudify/migrations/versions/272e61bf5f4a_6_3_to_6_4.py:45]: Ah- I assumed alembic had good reason for doing both. I'll let it implicitly drop indices then.

File: rest-service/manager_rest/storage/management_models.py (hunk 2 above)
Comment (tehasdf) [reply rest-service/manager_rest/storage/management_models.py:488]: oh, right, it's a hash. Maybe `secret_hash`?

File: rest-service/manager_rest/storage/management_models.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (geokala) [reply rest-service/manager_rest/storage/management_models.py:484]: OK, will resolve that with the same commit introducing the other columns.

Comment (geokala) [reply rest-service/manager_rest/storage/management_models.py:484]: Yup, adding.

File: resources/rest-service/cloudify/migrations/versions/272e61bf5f4a_6_3_to_6_4.py (hunk 3 above)
Comment (tehasdf) [reply resources/rest-service/cloudify/migrations/versions/272e61bf5f4a_6_3_to_6_4.py:45]: Nah it's just always emitting drops for corresponding creates. It doesn't have an optimizer

File: rest-service/manager_rest/storage/management_models.py (hunk 2 above)
Comment (geokala) [reply rest-service/manager_rest/storage/management_models.py:488]: Sure, why not

**Review by tehasdf — APPROVED** [review]

**Pull request merged**